pkg/controller/kubedirectorcluster/members.go

```diff
@@ -444,7 +444,6 @@ func handleCreatingMembers(
 	allRoles []*roleInfo,
 	configmetaGenerator func(string) string,
 ) {
-
```

Comment:
Let's put this blank line back (for code format consistency).

Reply:
Done